### ISSUE TITLE
React Integration

### ISSUE THREAD

**Issue by AlanFoster** [marionebl/svg-term#5]
Hey; Great work on this player. I wonder if it's possible to integrate it directly with React?

**Comment by marionebl** [comment]
Good idea - I am open to expose the [React component](https://github.com/marionebl/svg-term/blob/master/src/index.tsx#L91) used to generate the output string as a reusable export. 

If somebody is interested a PR would be very welcome.
